binder/requirements.txt

```diff
@@ -1,8 +1,9 @@
 matplotlib
 networkx>=2.0,<2.7
 numpy
-git+https://github.com/ObieCRE/openpyxl@b2edc9#egg=openpyxl-3.0.10.obie.2
+git+https://github.com/ObieCRE/openpyxl@b2edc9#egg=openpyxl-3.0.10+obie.2
 python-dateutil
 pydot
 ruamel.yaml
-freezegun
\ No newline at end of file
+freezegun
+ipdb
```

Comment:
helpful debug library, guessing there is better place to put this dependency wise, but not sweating for now